# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../plptools/64bit-int-fix.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Some 64bit compile fixes. Plpftp works on x86-64, plpnfsd not.

diff -Naur plptools-0.12.orig/include/plp_inttypes.h plptools-0.12/include/plp_inttypes.h
--- plptools-0.12.orig/include/plp_inttypes.h	2002-02-14 20:56:41.000000000 +0200
+++ plptools-0.12/include/plp_inttypes.h	2005-03-24 20:59:34.822833400 +0200
@@ -27,7 +27,11 @@
 typedef unsigned char      u_int8_t;
 typedef unsigned short     u_int16_t;
 typedef unsigned int       u_int32_t;
+#ifdef __x86_64__
+typedef long unsigned int u_int64_t;
+#else
 typedef unsigned long long u_int64_t;
+#endif
 typedef signed long long   s_int64_t;
 #endif // ISO_INTTYPES
 #endif // GNU_INTTYPES
